### PR TITLE
Open module containing last viewer content

### PR DESCRIPTION
**Pull request by ChrisKraljevicD2L**
I ain't gonna be able to test this for a while since I gotta rebuild my LMS, but this is the gist of the changes

### CODE REVIEW

File: d2l-sequence-launcher-unit/d2l-sequence-launcher-module.js [hunk 1]
```diff
@@ -491,5 +494,12 @@ class D2LSequenceLauncherModule extends ASVFocusWithinMixin(PolymerASVLaunchMixi
 	_getHideDescriptionClass(_hideModuleDescription, isSidebar) {
 		return _hideModuleDescription && !isSidebar ? 'hide-description' : '';
 	}
+
+	_openModule(moduleStartOpen) {
```
Comment (ianlaidlaw) [inline d2l-sequence-launcher-unit/d2l-sequence-launcher-module.js]: nit: for consistency, this should be kept as `_moduleStartOpen` I think

File: d2l-sequence-launcher-unit/d2l-sequence-launcher-module.js [hunk 2]
```diff
@@ -491,5 +494,12 @@ class D2LSequenceLauncherModule extends ASVFocusWithinMixin(PolymerASVLaunchMixi
...
+		if (moduleStartOpen) {
+			this.shadowRoot.querySelector('d2l-labs-accordion-collapse').setAttribute('opened', '');
+			this._moduleWasExpanded = true;
```
Comment (ianlaidlaw) [inline d2l-sequence-launcher-unit/d2l-sequence-launcher-module.js]: I might be wrong, but I don't think we need to set this but I suppose it doesn't hurt

Comment (ChrisKraljevicD2L) [reply d2l-sequence-launcher-unit/d2l-sequence-launcher-module.js]: I _think_ you're right, but it is used in `_getShowModuleChildren` so I'd rather set it to true to be extra safe

Comment (ChrisKraljevicD2L) [reply d2l-sequence-launcher-unit/d2l-sequence-launcher-module.js]: Oh it's `OR`'d. I stand corrected

**Pull request merged**